src/stdatamodels/jwst/datamodels/schemas/core.schema.yaml

```
@@ -1075,6 +1075,46 @@ properties:
             type: integer
             fits_keyword: DETYSIZ
             blend_table: True
+          multistripe_reads1:
+            title: Num. substripe rows read starting at SUBSTRT1
+            type: number
+            fits_keyword: MSTR_RD1
+            blend_table: True
+          multistripe_skips1:
+            title: Num. rows to skip after reading MSTR_RD1 rows
+            type: number
+            fits_keyword: MSTR_SK1
+            blend_table: True
+          multistripe_reads2:
+            title: Num. rows read in subsequent substripes
+            type: number
+            fits_keyword: MSTR_RD2
+            blend_table: True
+          multistripe_skips2:
+            title: Num. rows to skip between susequent substripes
+            type: number
```

```suggestion
            title: Number of rows to skip between subsequent substripes
            type: number
```
Fixed spelling and should this be an integer?